src/models/CollectionFlow.ts

```diff
@@ -0,0 +1,12 @@
+import { CollectionItem } from '@/models/CollectionItem'
+
+export type CollectionFlow = CollectionItem<'flow'> & {
```

Comment:
I see these fields on the `CollectionFlowResponse` type but not here.
The mapper has `pathContainingFlow` and `repositoryUrl` but not `entrypoint`

```suggestion
export type CollectionFlow = CollectionItem<'flow'> & {
entrypoint: string,
pathContainingFlow: string,
repositoryUrl: string,
```

Reply:
I've rolled back some of the complexity of these types. See [here](https://github.com/PrefectHQ/prefect-ui-library/pull/1144#discussion_r1103297542).

Take another look at the latest and let me know if you still have questions

Reply:
yep, looks good